apps/design-system/src/components/OperationIcon.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { OperationIcon } from 'ui';
+
+export default {
+    title: 'Components/OperationIcon',
+    component: OperationIcon,
+};
+
+export const WithReplyIcon = () => <OperationIcon operation="reply" className="w-7 h-7 fill-current text-gray-500" />;
```

Comment:
Any reason you have fill-current and text-gray-500 classes?

Reply:
No reason for that 😅